_includes/sidebar.html

```diff
@@ -9,6 +9,8 @@
       <li>
         <a href="{{folder.external_url}}" target="_blank">{{ folder.title }}</a>
       </li>
+      {% elsif folder.spacer %}
+      <br>
```

Comment:
I'll take a crack at it. Would you mind opening an issue for me? You can merge this and assign the other issue to me. Thanks!